README.md

```diff
@@ -3,3 +3,92 @@ An experimental RESTful storage API with various pluggable backends.
 
 The initial implementation targets elastic search, planned are mongodb, possibly dynamodb
 
+
+##How to use ElasticSearch storage API
+
+up-restorage.exe  --port=9300 --id-map="people:uuid,organisations:uuid" elastic  --index-name="concepts"  http://localhost:9200/  
+up-restorage.exe  --id-map="people:uuid,organisations:uuid" elastic  --index-name="concepts"  http://localhost:9200/  
+_Caviet: the order of args is important, swapping args order will  fail to start the app_
+
+### Single Document endpoints usage
+PUT http://localhost:8765/organisations/013f7fa7-aa26-3e20-84f1-fb8e5f7383ff  
+```
+{
+"uuid": "013f7fa7-aa26-3e20-84f1-fb8e5f7383ff",
+"properName": "Barclays Plc",
+"identifiers": [
+{
+"authority": "http://api.ft.com/system/FACTSET-EDM",
+"identifierValue": "05H4C8-E"
+},
+{
+"authority": "http://api.ft.com/system/FT-UPP",
+"identifierValue": "013f7fa7-aa26-3e20-84f1-fb8e5f7383ff"
+},
+{
+"authority": "http://api.ft.com/system/LEI",
+"identifierValue": "213800LBQA1Y9L22JB70"
+},
+{
+"authority": "http://api.ft.com/system/FT-TME",
+"identifierValue": "TnN0ZWluX09OX0ZvcnR1bmVDb21wYW55X0JBUkM=-T04="
+}]
+}
+```
+
+GET http://localhost:8080/organisations/013f7fa7-aa26-3e20-84f1-fb8e5f7383ff  
+DELETE http://localhost:8080/organisations/013f7fa7-aa26-3e20-84f1-fb8e5f7383ff  
+
+### Bulk Document endpoints usage
+PUT http://localhost:8080/people/  
+```
```

Comment:
We should put data in a public repository if it is derived from factset data.  I don't believe our contract allows it.